types/react-timeout/index.d.ts

```diff
@@ -1,5 +1,3 @@
-/// <reference types="node" />
```

Comment:
React packages should not be force-including @types/node for consumers. The timer types (Node.js objects or DOM integers) can be inferred.